components/arm/wrapper/wrapper.go

```diff
@@ -24,19 +22,19 @@ import (
 
 // AttrConfig is used for converting config attributes.
 type AttrConfig struct {
-	ModelPath string `json:"model-path"`
-	ArmName   string `json:"arm-name"`
+	ModelFilePath string `json:"model-path"`
+	ArmModel      string `json:"arm-model"`
```

Comment:
This should stay `ArmName`. Unlike `ModelFilePath`, which means the same thing as on `fake`, `ArmName` here is *not* the same use/meaning as `ArmModel` on `fake`.